src/SwedbankPay.Sdk/PaymentInstruments/Vipps/IVippsPaymentsResource.cs

```diff
@@ -1,25 +1,25 @@
-﻿using System;
-using System.Threading.Tasks;
-
-namespace SwedbankPay.Sdk.Payments.VippsPayments
-{
-    public interface IVippsPaymentsResource
-    {
-        /// <summary>
-        ///     Creates a new Vipps payment
-        /// </summary>
-        /// <param name="paymentRequest"></param>
-        /// <param name="paymentExpand"></param>
-        /// <returns></returns>
-        Task<VippsPayment> Create(VippsPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);
-
-
-        ///// <summary>
-        /////     Gets an existing Vipps payment.
-        ///// </summary>
-        ///// <param name="id"></param>
-        ///// <param name="paymentExpand"></param>
-        ///// <returns></returns>
-        Task<VippsPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None);
-    }
-}
+﻿using System;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Vipps
+{
+    public interface IVippsResource
+    {
+        /// <summary>
+        ///     Creates a new Vipps payment
+        /// </summary>
+        /// <param name="paymentRequest"></param>
+        /// <param name="paymentExpand"></param>
+        /// <returns></returns>
+        Task<IVippsPaymentReponse> Create(VippsPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)